workspace.json

```diff
@@ -5249,7 +5249,13 @@
             "cwd": "apps/judicial-system/backend"
           }
         },
-        "docker-express": {}
+        "docker-express": {},
+        "extract-strings": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "command": "yarn ts-node -P libs/localization/tsconfig.lib.json libs/localization/scripts/extract 'apps/judicial-system/backend/src/app/messages/**/*.{js,ts,tsx}'"
```

Comment:
I guess you can make this path stricter to either target the `requestPdf.ts` directly or just the `*.ts` within the folder

Reply:
Yes, definitely! I'll remove the js and tsx but we plan on having more translations on the messages folder so I want to target the whole folder 😃